test/e2e/crd_e2e_test.go

```diff
@@ -18,6 +18,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	longtimeout = time.Second * 75 // InstallPlan timeout is 60 seconds. This timeout is longer than that to get the "InstallPlanPhaseFailed" status
```

Comment:
Can we move the comment about the variable here? Maybe change the name to `installPlanTestTimeout`?

Reply:
Thanks for review.  Sure, I'll make changes.